examples/simple/outputs.tf

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "topic" {
+  value       = "${module.pubsub.topic}"
+  description = "The name of the Pub/Sub topic"
+}
+
+output "id" {
+  value       = "${module.pubsub.id}"
+  description = "The ID of the Pub/Sub topic"
+}
+
+output "uri" {
+  value       = "${module.pubsub.uri}"
+  description = "The URI of the Pub/Sub topic"
+}
+
+output "subscriptions" {
+  value = "${module.pubsub.subscriptions}"
+
+  description = "The name list of Pub/Sub subscriptions"
```

Comment:
The description must be updated to accurately reflect the contents of the value.

Reply:
Done.

Reply:
This description was missed in the change.

Reply:
```suggestion
  description = "A map containing the name and path key-value pairs of each created Pub/Sub subscription"
```